zebra-consensus/src/checkpoint.rs

```diff
@@ -412,6 +412,18 @@ impl CheckpointVerifier {
             .entry(height)
             .or_insert_with(|| QueuedBlockList::with_capacity(1));
 
+        let hash = block.as_ref().into();
+
+        for qb in qblocks.iter_mut() {
+            if qb.hash == hash {
+                let old_tx = std::mem::replace(&mut qb.tx, tx);
+                let e = "rejected older of duplicate verification requests".into();
+                tracing::warn!(?e);
```

Comment:
Question: Is a warn-level message appropriate here?

I don't think the syncer can guarantee that we won't have any duplicate blocks, because it might download duplicates after we restart and discard state. So let's info or debug instead.

Reply:
Yes, debug would be better, I'll change this in a fixup commit.